docs/layouts/shortcodes/generated/rest_v1_dispatcher.html

```diff
@@ -571,7 +571,7 @@
     "allowNonRestoredState" : {
       "type" : "boolean"
     },
-    "claimMode" : {
+    "restoreMode" : {
```

Comment:
Ah, don't know why is this changed? Caused by re-generating the documents?

Reply:
Sorry I made the mistake of manually modifying, after re-executing `generate-rest-docs`, `rest_v1_dispatcher.html`did not need to be modified, but' `est_v1_dispatcher.yml` did.